region/maridia/outer.json

```diff
@@ -4572,24 +4578,40 @@
               "id": 1,
               "strats": [
                 {
-                  "name": "Base",
+                  "name": "Gravity IBJ",
                   "notable": false,
                   "requires": [
-                    "Morph",
                     "Gravity",
-                    {"or": [
-                      "HiJump",
-                      "canWalljump",
-                      "h_canFly",
-                      "canSpringBallJumpMidAir"
-                    ]}
+                    "h_canIBJ"
                   ]
                 },
                 {
-                  "name": "Suitless Jump Assists",
+                  "name": "Shinespark",
                   "notable": false,
                   "requires": [
                     "Morph",
+                    {"canComeInCharged": {
+                      "fromNode": 2,
+                      "framesRemaining": 0,
```

Comment:
I can't think of a door that has a problem with a right side spark exit.

Reply:
I barely got a wasteland spark up left side.  But in that case you can diagonal spark, or would be able to gravity jump and the canLeaveCharged is hard enough to assume a lot of tech enabled.